Store/src/database/database.cs

```diff
@@ -130,30 +111,114 @@ PRIMARY KEY (id)
 
     public static async Task LoadPlayer(CCSPlayerController player)
     {
-        using MySqlConnection connection = Connect();
-
-        dynamic result = await connection.QueryFirstOrDefaultAsync(@"SELECT * FROM store_players WHERE SteamID = @SteamID",
-        new
+        if (player == null || !player.IsValid || player.IsBot)

```

Comment:
We don't need if check here because we already use it before using LoadPlayer.

Reply:
I'm sorry, I didn't look at it carefully